### PR TITLE
I Added The [Build|Passing] Tag! :smiley: 

### PR DESCRIPTION
**Pull request by jtrent238**
I Added The  Tag! :smiley:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# developer.github.com
+# developer.github.com ![](https://api.travis-ci.org/github/developer.github.com.svg)
```
Comment (Shredder121) [inline README.md]: If you make it this one it's clicking through to Travis.
```
[![Build Status](https://travis-ci.org/github/developer.github.com.svg?branch=master)](https://travis-ci.org/github/developer.github.com)
```

**Comment by mikemcquaid** [comment]
Thanks for the PR but we're not interested in this tag at this time. Have a great day!

**Comment by mikemcquaid** [comment]
@Shredder121 Can you describe your use-case? Thanks!

**Comment by jtrent238** [comment]
@Shredder121 Yea this is better, because I think the one I had was static, but not sure if it was.

 [Build Status](https://travis-ci.org/github/developer.github.com)
Add a line note

`[Build Status](https://travis-ci.org/github/developer.github.com)
Add a line note
`

**Comment by Shredder121** [comment]
@mikemcquaid Every now and then I visit the [branches overview of Travis](https://travis-ci.org/github/developer.github.com/branches).
This lists in a nutshell the few latest builds on master. (which indicate the stability of the build)
For example, if it weren't for commit 8b0528e0381cd00918e5c0c1a4c579a5795ca549, I'd have submitted a PR for it.

Basically, to get to the branches overview, you either have to bookmark it, or you can click through to it via a PR, but to get to a PR it requires a lot of clicks counting from the GitHub Homepage.

- Click the repo in the 'Repositories you contribute to' panel
- Click the `pull requests` tab
- Pick one (maybe there are no open ones)
- Scroll to pull request status, click travis build
- Click `branches` tab

Compare that to:

- Click the repo in the 'Repositories you contribute to' panel
- Click the Travis badge (which already shows an indication of the stability)
- Click `branches` tab

Even more actions when on mobile, but it happens.

I just appreciate developer resources to be in a common place. Even developer resources for a developer resource.

I hope any of this makes sense.

**Comment by mikemcquaid** [comment]
Ok, seems reasonable, thanks!

**Comment by mikemcquaid** [comment]
@jtrent238 Thanks for your contribution. You rock!

**Comment by Shredder121** [comment]
@jtrent238 Yes you rock!

**Pull request merged**